packages/auth/src/graphql/resolvers/grant.test.ts

```diff
@@ -481,38 +482,46 @@ describe('Grant Resolvers', (): void => {
     })
 
     test('Returns error for unknown grant', async (): Promise<void> => {
-      const gqlQuery = appContainer.apolloClient
-        .mutate({
-          mutation: gql`
-            query GetGrant($id: String!) {
-              grant(id: $id) {
-                id
-                client
-                state
-                access {
+      try {
+        const gqlQuery = appContainer.apolloClient
+          .mutate({
+            mutation: gql`
+              query GetGrant($id: ID!) {
+                grant(id: $id) {
                   id
-                  identifier
+                  client
+                  state
+                  access {
+                    id
+                    identifier
+                    createdAt
+                    actions
+                    type
+                  }
                   createdAt
-                  actions
-                  type
                 }
-                createdAt
               }
+            `,
+            variables: {
+              id: uuid()
             }
-          `,
-          variables: {
-            id: uuid()
-          }
-        })
-        .then((query): Grant => {
-          if (query.data) {
-            return query.data.grant
-          } else {
-            throw new Error('Data was empty')
-          }
-        })
+          })
+          .then((query): Grant => {
+            if (query.data) {
+              return query.data.grant
+            } else {
+              throw new Error('Data was empty')
+            }
+          })
 
-      await expect(gqlQuery).rejects.toThrow(ApolloError)
+        await expect(gqlQuery).rejects.toThrow(ApolloError)
+        await expect(gqlQuery).rejects.toThrow('No grant')
```

Comment:
would it makes sense to just do these assertions in the catch?

Reply:
It would still work, and I personally think it's cleaner to have one level of assertion, rather than the assertion-of-assertions approach of https://github.com/interledger/rafiki/pull/2676#discussion_r1586408057